### PR TITLE
OLED color theme fixes, UI alignment fixes, community insert fix

### PR DESCRIPTION
**Pull request by CTalvio**
## Pull Request Description

This is a collection of tiny coloring and UI changes

## Issue Being Fixed

The OLED theme has several places where things go awry if it is used together with material you. This looks to change theming so that material you colors are applied correctly.

I also did some small alignment fixes, and removed an extraneous divider that only shows up on ones own profile.

Maybe could have been a separate PR, but lastly this changes community insert action to simply pasting the ! format plaintext name of the community, as this is clickable in most clients without taking users off-instance. The insert also erroneously used an @ instead of an !.

## Screenshots / Recordings

image
image
image
image
image

## Checklist

- [ ] Did you update CHANGELOG.md?
- [ ] Did you use localized strings where applicable?
- [ ] Did you add `semanticLabel`s where applicable for accessibility?

### CODE REVIEW

**Comment by micahmo** [comment]
> image

This should address one of @codenyte's issues [here](https://github.com/thunder-app/thunder/issues/1321#issue-2254604069).

**Comment by CTalvio** [comment]
@codenyte I'll likely be dailying the oled theme when this gets fixed. I want to use it but the various bugs in it bothered me too much.

Hopefully with my eyes on it it'll be better going forward.

File: lib/comment/view/create_comment_page.dart [hunk 1]
```diff
@@ -407,7 +406,7 @@ class _CreateCommentPageState extends State<CreateCommentPage> {
                                   MarkdownType.community: () {
                                     showCommunityInputDialog(context, title: l10n.community, onCommunitySelected: (community) {
                                       _bodyTextController.text = _bodyTextController.text.replaceRange(_bodyTextController.selection.end, _bodyTextController.selection.end,
-                                          '[@${community.community.title}@${fetchInstanceNameFromUrl(community.community.actorId)}](${community.community.actorId})');
+                                          '!${community.community.name}@${fetchInstanceNameFromUrl(community.community.actorId)}');
```
Comment (micahmo) [inline lib/comment/view/create_comment_page.dart:409]: Should we remove the URL for users too?

**Review by micahmo — APPROVED** [review]
LGTM!

**Comment by CTalvio** [comment]
I'm going to leave this PR unchanged to get it merged, but feel free to open an issue and list as many spots in the OLED theme that need fixing as you can find for me to address in a future PR.

**Review by hjiangsu — APPROVED** [review]
LGTM! I'll run a build with these changes to see if I notice anything off with dark mode (since that's the one I usually use)

**Pull request merged**